account_invoice_merge/models/account_invoice.py

```diff
@@ -41,7 +41,7 @@ def _get_invoice_line_key_cols(self):
             'product_id', 'account_id', 'account_analytic_id',
             'uom_id'
         ]
-        for field in ['analytics_id']:
+        for field in ['analytics_id', 'sale_line_ids']:
```

Comment:
The 'analytics_id' field was added by `account_analytic_plans` base module. But this base module was removed in v9. So this field is not anymore in versions 9.0 and then here it should be removed.